package.json

```diff
@@ -148,8 +148,8 @@
     "css-loader": "^0.28.10",
     "decompress": "^4.2.0",
     "enhanced-resolve": "^0.7.6",
-    "enzyme": "^3.1.0",
-    "enzyme-adapter-react-16": "^1.7.1",
+    "enzyme": "^3.11.0",
```

Comment:
had to upgrade this stuff because older versions of Enzyme did not properly do full mounting of components that users React Fragments. I did not actually need to upgrade `enzyme-adapter-react-16` but why not

Reply:
@jhonnyoddball fyi